### PR TITLE
r.buildvrt: document performance issues with external data

### PR DESCRIPTION
**Pull request by ninsbl**
This PR aims at documenting the known performance issues of GRASS VRTs with external, linked data.

Not sure, if or how to mention the [r.buildvrt.gdal](https://grass.osgeo.org/grass84/manuals/addons/r.buildvrt.gdal.html) addon?

Fix: #4345

### CODE REVIEW

File: raster/r.buildvrt/r.buildvrt.html [hunk 1]
```diff
@@ -48,6 +50,15 @@ <h3>VRT from a DEM in the North Carolina sample dataset</h3>
...
 </pre></div>
 
+<h2>KNOWN ISSUES</h2>
+Users may experience significant performance degradation with virtual rasters build
+with <em>r.buildvrt</em> over GDAL-linked (<em>r.external</em>) raster maps, especially
+with slower file systems with latency like NFS. Performance degradation can also be
+noticable on local file systems, but is usually less severe.
+<p>
+For GDAL-linked raster maps it is thus recommended to build GDAL VRTs, e.g with
+<em>gdalbuildvrt</em>.
```
Comment (neteler) [inline raster/r.buildvrt/r.buildvrt.html:60]: Could you pls make it "https" for GDAL below? Cannot reach it from the web editing interface.

**Comment by wenzeslaus** [comment]
> Not sure, if or how to mention the [r.buildvrt.gdal](https://grass.osgeo.org/grass84/manuals/addons/r.buildvrt.gdal.html) addon?

Definitively mention it. And also add it to "See also" too with full URL to grass-stable.

File: raster/r.buildvrt/r.buildvrt.html (hunk 1 above)
Comment (ninsbl) [reply raster/r.buildvrt/r.buildvrt.html:62]: See: [223e8e5](https://github.com/OSGeo/grass/pull/4441/commits/223e8e5a10547a091acdde45f741f5c66f01fb11)

**Comment by ninsbl** [comment]
Comments addressed in [223e8e5](https://github.com/OSGeo/grass/pull/4441/commits/223e8e5a10547a091acdde45f741f5c66f01fb11)

File: raster/r.buildvrt/r.buildvrt.html [hunk 2]
```diff
@@ -12,6 +12,8 @@ <h2>NOTES</h2>
 the original raster maps which is only valid if the original raster
 maps remain in the originally indicated mapset. A VRT can also be built
 from raster maps registered with <em><a href="r.external.html">r.external</a></em>.
+However, there are known performance issues with GRASS VRTs build
+over external, registered data (see below).
```
Comment (neteler) [inline raster/r.buildvrt/r.buildvrt.html:16]:
```suggestion
However, GRASS VRTs built from external registered data (see below)
are known to have performance issues.
```

File: raster/r.buildvrt/r.buildvrt.html [hunk 3]
```diff
@@ -48,18 +50,30 @@ <h3>VRT from a DEM in the North Carolina sample dataset</h3>
...
+<h2>KNOWN ISSUES</h2>
+Users may experience significant performance degradation with virtual rasters build
+with <em>r.buildvrt</em> over GDAL-linked (<em>r.external</em>) raster maps, especially
+with slower file systems with latency like NFS. Performance degradation can also be
+noticable on local file systems, but is usually less severe. For such use-cases
+consider using the GRASS GIS addon
+<a href="https://grass.osgeo.org/grass-stable/manuals/addons/r.buildvrt.gdal.html">r.buildvrt.gdal</a>.
+<p>
+For GDAL-linked raster maps it is thus recommended to build GDAL VRTs, e.g with
+<em>gdalbuildvrt</em>.
```
Comment (neteler) [inline raster/r.buildvrt/r.buildvrt.html:62]:
```suggestion
<h2>KNOWN ISSUES</h2>

Users may experience significant performance degradation with virtual rasters built
with <em>r.buildvrt</em> over GDAL-linked (<em>r.external</em>) raster maps,
especially on slower file systems with latency like NFS. Performance degradation
may also occur on local file systems, but is usually less severe. For such use cases
consider using the GRASS GIS addon
<a href="https://grass.osgeo.org/grass-stable/manuals/addons/r.buildvrt.gdal.html">r.buildvrt.gdal</a>.
<p>
For GDAL-linked raster maps it is thus recommended to build GDAL VRTs, e.g. with
<em>gdalbuildvrt</em>.
```

**Comment by ninsbl** [comment]
Thanks for the review @neteler Suggestions are now included. I also condensed the recommendation for GDAL-linked raster maps.

File: raster/r.buildvrt/r.buildvrt.html [hunk 4]
```diff
@@ -48,18 +50,29 @@ <h3>VRT from a DEM in the North Carolina sample dataset</h3>
...
 
+<h2>KNOWN ISSUES</h2>
+
+Users may experience significant performance degradation with virtual rasters built
+with <em>r.buildvrt</em> over GDAL-linked (<em>r.external</em>) raster maps,
+especially on slower file systems with latency like NFS. Performance degradation
+may also occur on local file systems, but is usually less severe. For such use cases
+consider using the GRASS GIS addon
+<a href="https://grass.osgeo.org/grass-stable/manuals/addons/r.buildvrt.gdal.html">r.buildvrt.gdal</a>.
+or building GDAL VRTs, e.g. with <em>gdalbuildvrt</em>.
```
Comment (neteler) [inline raster/r.buildvrt/r.buildvrt.html:61]:
```suggestion
<a href="https://grass.osgeo.org/grass-stable/manuals/addons/r.buildvrt.gdal.html">r.buildvrt.gdal</a>
or building GDAL VRTs, e.g. with <em>gdalbuildvrt</em>.
```

**Review by echoix — APPROVED** [review]

**Pull request merged**